src/store/modules/user.js

```diff
@@ -71,6 +75,25 @@ const actions = {
     commit("setName", name);
   },
 
+  async getSettings({ commit }) {
+    const settings = await API.get(
+      `${process.env.VUE_APP_MANAGER_API_URL}/v1/account/settings`
+    );
+
+    commit("setSettings", settings);
+  },
+
+  async updateSetting({ commit }, { setting, value }) {
+    const { data: settings } = await API.post(
+      `${process.env.VUE_APP_MANAGER_API_URL}/v1/account/settings/update`,
```

Comment:
```suggestion
      `${process.env.VUE_APP_MANAGER_API_URL}/v1/account/settings`,
```
The route has been changed [here](https://github.com/getumbrel/umbrel-manager/pull/58/commits/df5d150fd56040d9c0ec33fdcdb764c9f2e5d865).